tests/unit/completion/test_histcategory.py

```diff
@@ -133,6 +135,18 @@ def test_set_pattern_repeated(model_validator, hist):
     ])
 
 
+def test_set_pattern_long(hist, message_mock, caplog):
+    hist.insert({'url': 'example.com/foo', 'title': 'title1', 'last_atime': 1})
+    cat = histcategory.HistoryCategory()
+    with caplog.at_level(logging.ERROR):
+        # pylint: disable=bad-builtin
+        cat.set_pattern(" ".join(map(str, range(10000))))
```

Comment:
You could just replace `map(str, range(10000)))` with `str(x) for x in range(10000)` so you don't need the `pylint` disables.